components/ntp_background_images/browser/url_constants.cc

```diff
@@ -10,5 +10,11 @@ namespace ntp_background_images {
 const char kBrandedWallpaperHost[] = "branded-wallpaper";
 const char kLogoPath[] = "logo.png";
 const char kWallpaperPathPrefix[] = "wallpaper-";
+const char kSuperReferralPath[] = "super-referral/";
+const char kSponsoredImagesPath[] = "sponsored-images/";
+const char kSuperReferralMappingTableURL[] =
+    "https://mobile-data.s3.brave.com/superreferrer/map-table.json";
+const char kSuperReferralMappingTableDevURL[] =
+    "https://mobile-data-dev.s3.brave.software/superreferrer/map-table.json";
```

Comment:
may need to add these to https://github.com/brave/brave-browser/blob/master/lib/whitelistedUrlPrefixes.js so network audit passes

Reply:
Thanks for checking! Pushed https://github.com/brave/brave-browser/pull/9084